src/bp/ui-admin/src/Pages/Bot/Details.jsx

```diff
@@ -303,10 +337,19 @@ class Bots extends Component {
   }
 
   renderDetails() {
+    const {
+      categories,
+      category,
+      description,
+      error,
+      name,
+      status,
+      successMsg
+    } = this.state;
     return (
       <div>
-        {this.state.error && <Alert color="danger">{this.state.error.message}</Alert>}
-        {this.state.successMsg && <Alert type="success">{this.state.successMsg}</Alert>}
+        {error && <AlertBanner type="error" hide={() => this.setState({ error: null })}>{error.message}</AlertBanner>}
+        {successMsg && <AlertBanner type="success" hideCloseBtn={true} hide={() => this.setState({ successMsg: '' })}>{successMsg}</AlertBanner>}
```

Comment:
I like the way your alert banner looks ! However we use Blueprint's toast mechanism everywhere else, so it may seem a bit weird to have different info mechanism (there is the toastSuccess and toastFailure methods already available)

![image](https://user-images.githubusercontent.com/42552874/72670539-48958b00-3a0c-11ea-9baa-a141619cfda2.png) 
![image](https://user-images.githubusercontent.com/42552874/72670547-6105a580-3a0c-11ea-98cc-1f0e39e4049f.png)

Reply:
I was trying to replace the use of `<Alert />` from react-bootstrap which isn't sticky so it would show only at the top of the form and require to scroll back to see it if you were far down the form with all the collapsible open. Not sure if the Toast would or should be used for that purpose 🤔it's probably not a big deal either way I guess

Reply:
After thinking about it, I think I'm just not used to use Toasts, but It does serve the exact same purpose 😬 I'll change it